css/styles.css

```diff
@@ -76,6 +76,9 @@
   max-width: 300px;
   display: inline-block;
   animation: slide-up 0.5s ease;
+  box-shadow: 4px 7px 0 -1px rgba(214, 198, 198, 0.17);
+  -webkit-box-shadow: 4px 7px 0 -1px rgba(214, 198, 198, 0.17);
+  -moz-box-shadow: 4px 7px 0 -1px rgba(214, 198, 198, 0.17);
```

Comment:
Instead of adding these lines to a whole bunch of rules, you should make this its own class that you can reuse and apply to all these messages. You can name it something like `message-shadow`